pom.xml

```diff
@@ -97,13 +97,13 @@
     <dependency>
       <groupId>org.mock-server</groupId>
       <artifactId>mockserver-client-java</artifactId>
-      <version>5.3.0</version>
+      <version>5.4.1</version>
     </dependency>
     <!-- https://mvnrepository.com/artifact/org.mock-server/mockserver-netty -->
     <dependency>
       <groupId>org.mock-server</groupId>
       <artifactId>mockserver-netty</artifactId>
-      <version>5.3.0</version>
+      <version>5.4.1</version>
```

Comment:
Is these new versions FOSS approved?

Reply:
no, with him can i approve this?

Reply:
who is him?
I tried to find it in Bazaar,, but can't find any info regarding org.mock-server.

Reply:
I can't find the previous version either,, so we can merge this for the moment. We need to check this later.

Reply:
sorry with whom

Reply:
Ok, I found now that version 5.3.0 is requested in Bazaar,, but we need to request for version 5.4.1 later.

Reply:
But you can merge now and handle this later.